src/field_theory/tower.lean

```diff
@@ -86,16 +115,14 @@ by { rw [submodule.restrict_scalars_top, eq_top_iff, ← hb, submodule.span_le],
   exact submodule.subset_span }⟩⟩
 
 /-- Tower law: if `A` is a `K`-algebra and `K` is a field extension of `F` then
-`dim_F(A) = dim_F(K) * dim_K(A)`. -/
-theorem finrank_mul_finrank [finite_dimensional F K] :
-  finrank F K * finrank K A = finrank F A :=
+`dim_F(A) = dim_F(K) * dim_K(A)`.
```

Comment:
Same here.

Reply:
I've left this one alone since here we're still talking about vector spaces and not modules.